CIP-????/README.md

```diff
@@ -0,0 +1,471 @@
+---
+CIP: ????
+Title: Sums-of-products in Plutus Core
+Authors: Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Status: Proposed
+Category: Plutus
+Created: 2023-01-30
+License: CC-BY-4.0
+---
+
+# CIP-XXX: Sums-of-products in Plutus Core
+
```

Comment:
```suggestion
```
@michaelpj since your last round of CIP's we finally resolved the ambivalent treatment of H1 tags in the CIP markup... we're asking proposers not to include them, so the H1 titles can be generated automatically from the header data (note just added with https://github.com/cardano-foundation/CIPs/pull/450): https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#structure